### DISCUSSION TITLE
Individual numbers are recognized correctly but the resulting value is wrong

### DISCUSSION THREAD

**Discussion by TomTomB** [jomjol/AI-on-the-edge-device#2959] [Q&A] [answered]
On the recognition page, everything looks fine, but this results in a raw value of 02995.8396 (should be 02994.8396). Am I missing something? Is there maybe a setting for that?
<img width="769" alt="Screenshot 2024-02-29 234649" src="https://github.com/jomjol/AI-on-the-edge-device/assets/12258325/90a5785a-d9b1-492a-9757-7650b47c8e61">

**Comment by Slider0007** [comment, marked as answer]
You have to lower the expert parameter AnalogDigitalTransistionStart to roughly 7.5 (parameter value needs to be lower than the 0.1x analog pointer value when last digit just reaches x.8).
